oxidation/client/android/app/build.gradle

```diff
@@ -1,6 +1,8 @@
 apply plugin: 'com.android.application'
 
 android {
+    namespace "com.scille.parsec"
```

Comment:
What is this for ?

Reply:
For this error message

```
package="com.scille.parsec" found in source AndroidManifest.xml: /home/andesite/Documents/parsec-cloud/oxidation/client/android/app/src/main/AndroidManifest.xml.
Setting the namespace via a source AndroidManifest.xml's package attribute is deprecated.
Please instead set the namespace (or testNamespace) in the module's build.gradle file, as described here: https://developer.android.com/studio/build/configure-app-module#set-namespace
```

tl,dr: we need to specify the namespace in `build.gradle`